crates/matrix-sdk-crypto/src/file_encryption/key_export.rs

```diff
@@ -75,19 +75,19 @@ pub enum KeyExportError {
 ///
 /// * `passphrase` - The passphrase that was used to encrypt the exported keys.
 ///
-/// # Examples
+/// # Example
```

Comment:
Hmm, are you changing this because you prefer the `Example` section name project-wide or because this specific section has a single example?

Reply:
There's a single example here so it seemed more natural, and we weren't consistent anyways. But I'm happy to revert this and use "Examples" consistently, looks like that's what the standard library does.

Reply:
Yeah, being consistent with section names sounds better to me as well.